### ISSUE TITLE
Not work for "sd-webui-reactor" extension

### ISSUE THREAD

**Issue by makisekurisu-jp** [shirayu/sd-webui-enable-checker#57]
## Describe the bug

![20230920175529](https://github.com/shirayu/sd-webui-enable-checker/assets/72290820/cd828d7e-fa6f-45fc-8fc1-a80626b9b4d4)

## Logs (Optional)
```
venv "D:\AITech\stable-diffusion-webui\venv\Scripts\Python.exe"
Python 3.10.11 (tags/v3.10.11:7d4cc5a, Apr  5 2023, 00:38:17) [MSC v.1929 64 bit (AMD64)]
Version: v1.6.0
Commit hash: 5ef669de080814067961f28357256e8fe27544f4
Checking ReActor requirements... Ok
Launching Web UI with arguments: --xformers --no-gradio-queue
Tag Autocomplete: Could not locate model-keyword extension, Lora trigger word completion will be limited to those added through the extra networks menu.
2023-09-20 17:49:00,392 - ControlNet - INFO - ControlNet v1.1.410
ControlNet preprocessor location: D:\AITech\stable-diffusion-webui\extensions\sd-webui-controlnet\annotator\downloads
2023-09-20 17:49:00,488 - ControlNet - INFO - ControlNet v1.1.410
sd-webui-prompt-all-in-one background API service started successfully.
17:49:01 - ReActor - STATUS - Running v0.4.2-b2
Loading weights [48a00220c1] from D:\AITech\stable-diffusion-webui\models\Stable-diffusion\None_HD_Portrait_v10.safetensors
Running on local URL:  http://127.0.0.1:7860
Creating model from config: D:\AITech\stable-diffusion-webui\configs\v1-inference.yaml

To create a public link, set `share=True` in `launch()`.
[Lobe]: Initializing Lobe
Startup time: 13.6s (prepare environment: 3.4s, import torch: 3.0s, import gradio: 1.6s, setup paths: 0.8s, initialize shared: 0.3s, other imports: 1.0s, setup codeformer: 0.1s, load scripts: 2.4s, create ui: 0.5s, gradio launch: 0.5s).
Applying attention optimization: xformers... done.
Model loaded in 12.3s (load weights from disk: 0.7s, create model: 4.6s, apply weights to model: 3.8s, apply half(): 1.2s, load VAE: 0.2s, calculate empty prompt: 1.7s).
```

## Environment
python: 3.10.11
torch: 2.0.1+cu118

- OS:win11pro 22h2
- Browser Version:edge v117.0.2045.36
- stable-diffusion-webui Version: v1.6.0
- sd-webui-enable-checker Version: c655cf1d

**Comment by shirayu** [comment]
@Amazon90 Hi, are you using a kind of Chinese translation feature?
While the label of the original checkbox seems to be `Enable`, one in the screenshot is in Chinese.
https://github.com/Gourieff/sd-webui-reactor/blob/fa2a7f4c3910ba487d1c8bc99c9fa807800d6568/scripts/faceswap.py#L48

**Comment by makisekurisu-jp** [comment]
> @Amazon90 Hi, are you using a kind of Chinese translation feature?
> 
> While the label of the original checkbox seems to be `Enable`, one in the screenshot is in Chinese.
> 
> https://github.com/Gourieff/sd-webui-reactor/blob/fa2a7f4c3910ba487d1c8bc99c9fa807800d6568/scripts/faceswap.py#L48

https://github.com/harukaxxxx/stable-diffusion-webui-localization-zh_TW

I think enable checker not work is not caused by localized language. Because this problem does not occur in other extensions.
![20230920202705](https://github.com/shirayu/sd-webui-enable-checker/assets/72290820/5c6e9340-23b7-4caa-92c0-1ffc9e91e012)

**Comment by shirayu** [comment]
Right.
I added "啟用" and "启用" to detect the "Enable" checkboxes.

- https://github.com/shirayu/sd-webui-enable-checker/commit/f5b883303880fb68ffb43c18b4f6c27914d58f6e
- https://github.com/harukaxxxx/stable-diffusion-webui-localization-zh_TW/blob/9ef7397f8badd391e47d41ed2a6146ed96f26d6b/localizations/zh_TW.json#L1377C14-L1377C16


Please try with the latest version.

**Comment by makisekurisu-jp** [comment]
> Right.
> 
> I added "啟用" and "启用" to detect the "Enable" checkboxes.
> 
> 
> 
> - https://github.com/shirayu/sd-webui-enable-checker/commit/f5b883303880fb68ffb43c18b4f6c27914d58f6e
> 
> - https://github.com/harukaxxxx/stable-diffusion-webui-localization-zh_TW/blob/9ef7397f8badd391e47d41ed2a6146ed96f26d6b/localizations/zh_TW.json#L1377C14-L1377C16
> 
> 
> 
> 
> 
> Please try with the latest version.

It's work now.Thank you!
